### PR TITLE
Create e2e test

### PR DESCRIPTION
**Pull request by lewismarshall**
Should probably do this on master only maybe

### CODE REVIEW

**Comment by lewismarshall** [comment]
So this is the first stab - it's not perfect but enough time has been spent for now and I've captured enhancements with issue #57

File: tests/e2e-container/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        --add-repo \
+        https://download.docker.com/linux/fedora/docker-ce.repo -y && \
+        dnf makecache fast -y && \
+        dnf install \
+        docker-ce \
+        -y  && \
+        dnf clean all
+
+ENV container docker
+#RUN rm /usr/lib/systemd/system/lvm2-lvmetad.service
```
Comment (vaijab) [inline tests/e2e-container/Dockerfile]: Not needed if commented out?

File: tests/e2e-container/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+RUN curl -s https://pkg.cfssl.org/R1.1/cfssl_linux-amd64 -o /usr/bin/cfssl && \
+    chmod +x /usr/bin/cfssl
+RUN curl -s https://pkg.cfssl.org/R1.1/cfssljson_linux-amd64 -o /usr/bin/cfssljson && \
+    chmod +x /usr/bin/cfssljson
+
+# We need a specific version to work with what's shipping in CoreOS
```
Comment (vaijab) [inline tests/e2e-container/Dockerfile]: How do you know what's shipping with CoreOS at any given time?

File: tests/e2e.sh [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+cd ${script_dir}/..
+
+function cleanup() {
+  docker stop ${name} || true
+  docker rm ${name} || true
+}
+
+# TODO: move to a service container when drone works https://github.com/UKHomeOffice/keto-k8/issues/54
+# Run tests in a container with privileged options (not part of drone spec - yet).
+#       -v /var/run/dbus:/var/run/dbus \
```
Comment (vaijab) [inline tests/e2e.sh]: Remove this line?

File: tests/e2e-container/Dockerfile (hunk 2 above)
Comment (lewismarshall) [inline tests/e2e-container/Dockerfile]: As discussed we can't at the moment -I've updated the comment to explain for now.

**Pull request merged**